src/templates/blog-post.js

```diff
@@ -145,6 +145,7 @@ class BlogPostTemplate extends React.Component {
           title={post.frontmatter.title}
           description={post.frontmatter.spoiler}
           slug={post.fields.slug}
+          pathname={`/blog${post.fields.slug}`}
```

Comment:
`post.fields.slug` starts with `/` ?

Reply:
yeah its a bit weird. it starts and ends with `/`